site/multiDim/MultiDim.tsx

```diff
@@ -87,8 +88,11 @@ export default function MultiDim({
         )
         return config.filterToAvailableChoices(choices).selectedChoices
     })
+    const [shouldPreserveTab, setShouldPreserveTab] = useState(false)
```

Comment:
A comment here to explain why we need this would probably be nice for future refactorors :)

Reply:
Good idea, added.